docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -4,34 +4,101 @@ and pull requests should be submitted there:
-- The xref:{doc-guides}/doc-reference.adoc[Quarkus documentation reference] for required syntax and other conventions.
+- The xref:{doc-guides}/doc-reference.adoc[Quarkus documentation reference] for the required syntax, preferred style, and other conventions.
 
-== Sources
+== Locate the source files for Quarkus docs
 
 - AsciiDoc files are in the `src/main/asciidoc` directory within the `docs` module of the {quarkus-base-url}/tree/main/docs[Quarkus GitHub repository].
 - Configuration documentation is generated from JavaDoc comments in Java source files.
 - Java, YAML, and other source files can also be xref:doc-reference.adoc#reference-source-code[referenced] by AsciiDoc files.
+- The link:https://quarkus.io/guides/[Quarkus documentation] menu page, also known as the doc index page, is sourced in the link:https://github.com/quarkusio/quarkusio.github.io[quarkusio.github.io] repository.
+
+
+== Create Quarkus content in AsciiDoc
+
+To ensure that your content shows up correctly on the link:https://quarkus.io/guides/[Quarkus documentation home page], use the following steps:
+
+. Decide on a content type that best fits the content that you are contributing.
++
+TIP: To help you decide, use the *Content types* section in xref:{doc-guides}/doc-reference.adoc[Quarkus documentation reference].
++
+. Go to the `src/main/asciidoc/_templates` directory, and make a copy of the relevant template for the content type you have chosen. Be sure to:
+** Use the filename syntax of`<category>-<titlekeyword>-<titlekeyword>-<content-type>.adoc`. For example, `security-basic-authentication-tutorial.adoc`.
+** Save the file to the root of the `asciidoc` folder in the `quarkus` repository.
+. Set the minimum required header information as outlined in the following example:
++
+[source,asciidoc]
+----
+[id="security-basic-authentication-howto"] <1>
+= Secure a Quarkus application with basic authentication <2>
+include::_attributes.adoc[] <3>
+:categories: security <4>
+----
++
+{important-caption}
+
+<1> Set the `id` value to be the same as the file name but without the extension.
+<2> For information about how to create a good title for the content type, see xref:doc-reference.adoc[Quarkus documentation reference].
+<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the ToC displays, and content renders in the website portal. 
+<4> Set at least one category to ensure that the content is findable on the link:https://quarkus.io/guides/[Quarkus documentation home page].
++
+. Optional: If you have renamed the filename of an existing AsciiDoc file, see xref:doc-contribute-docs-howto#redirect-an-existing-Quarkus-AsciiDoc-file[Redirect an existing Quarkus AsciiDoc file].
+
+For more information about the minimum header requirements, see xref:{doc-guides}/doc-reference.adoc[Quarkus documentation reference].
+
+== Retire and redirect an existing Quarkus AsciiDoc source file
```

Comment:
```suggestion
== Retire and redirect an existing Quarkus AsciiDoc source file
```
@rolfedh - When `security-openid-connect.adoc` is no longer on `main`, could you review and test out the following steps for how to set up a URL redirect?  Thanks :pray: